src/js/amo.js

```diff
@@ -37,14 +37,13 @@ t.render(async () => {
 
   for (let addon of results) {
     try {
-      console.log(addon);
       let template = document.querySelector("#addon");
       let clone = document.importNode(template.content, true);
 
       let name = addon.name ? addon.name["en-US"] || addon.name[addon.default_locale] || "unknown" : addon.slug;
 
       clone.querySelector(".name").textContent = name;
-      clone.querySelector(".img").src = addon.icon_url || "../images/addon.png";
+      clone.querySelector(".img").src = addon.icon_url || "../src/images/addon.png";
```

Comment:
Is the path correct here? I think `src` is only for packaging? It seems to me the images should also be in src.

Reply:
It needed to required the url of the image to tell parcel to compile it and convert to the production URL.